models/LineCont.yaml

```diff
@@ -0,0 +1,7 @@
+type: LineCont  # comment or string or block
+start: String
+end: String  # implicitly end of line.
+escape: String
+length: Integer  # Length after escapes are processed
+indent: Boolean
+nested: Boolean
```

Comment:
I am not aware of any line continuation that is nested, nor indented, nor has embedded escapes, and the `length` is always 1 in your examples so far, so adding that field is redundant, unless you know of a case which has a multiple-character line continuation operator.

Reply:
got it.